esx_service/vmdk_ops_test.py

```diff
@@ -245,9 +245,9 @@ def testPolicy(self):
                                             volume_kv.DISK_ALLOCATION_FORMAT: unit[3]})
             self.assertEqual(err == None, unit[2], err)
 
-            # clean up should fail if the created should have failed.
+            # clean up would succeed with #1084.
```

Comment:
we actually do refer to github issues and PRs in different place, so while it may be good or bad, it is certainly consistent (there are 30 cases like that in 17 files in our repo, including ./vendor)